modules/main.tf

```diff
@@ -0,0 +1,12 @@
+module "account" {
```

Comment:
it is supposed that this module call will be from external infra setups not from inside this "terraform-aws-account" module